orc8r/gateway/c/common/sentry/SentryWrapper.cpp

```diff
@@ -84,6 +85,7 @@ std::string get_snowflake() {
 }
 
 void initialize_sentry(
+    char node_name[50]
```

Comment:
where do we get this node_name from?

Reply:
oh i see, are you just trying to make space? in that case, can we move this down to line ~110 right before we call gethostname?

Reply:
yes, that would be clear.